src/SFA.DAS.DownloadService.Api/Controllers/AparController.cs

```diff
@@ -126,12 +130,23 @@ public async Task<IActionResult> GetLatestTime()
         {
             _log.LogDebug($"Fetching GET latest change date");
 
-            DateTime? latestChange = DateTime.UtcNow;
+            DateTime? latestChange = _dateTimeProvider.GetCurrentDateTime();
             try
             {
-                var result = await _roatpApiClient.GetLatestNonOnboardingOrganisationChangeDate();
-                if (result != null)
-                    latestChange = result;
+                var roatpResult = await _roatpApiClient.GetLatestNonOnboardingOrganisationChangeDate();
+                var aparResult = await _assessorApiClient.GetAssessmentOrganisationLastUpdated();
+
+                if (roatpResult != null && aparResult != null)
```

Comment:
This can be more clearly written as:

if (roatpResult.HasValue && aparResult.HasValue)
        {
            latestChange = roatpResult > aparResult ? roatpResult : aparResult;
        }
        else if (roatpResult.HasValue)
        {
            latestChange = roatpResult;
        }
        else if (aparResult.HasValue)
        {
            latestChange = aparResult;
        }

But then need to consider what a null means when returned from either of them in the original code a null return from RoATP results in the UtcDate being returned - so logically a null result from either of them should be the UtcDate so just:

if (roatpResult.HasValue && aparResult.HasValue)
        {
            latestChange = roatpResult > aparResult ? roatpResult : aparResult;
        }

Reply:
I have added the first code suggestion, the existing unit test passes with it, is that alright? I tried the second but the unit test failed when using null values